src/main/java/edu/oregonstate/cs361/battleship/Military.java

```diff
@@ -12,6 +12,28 @@ public Military(boolean b, String n, int l,Coordinate s, Coordinate e) {
         stealth = b;
     }
 
+    //New scan function that implements the stealth feature, original scan function is in Ship.java --Rebecca
+    public boolean scan(Coordinate coor) {
+        if (stealth == true) {
```

Comment:
Just kidding, this is probably the best way to do it inheritance-wise. Good work